.circleci/config.yml

```diff
@@ -12,12 +12,12 @@ commands:
   install-dependencies:
     steps:
```

Comment:
looks like we have circle and gha in here. Did we mean to keep one over the other?

Reply:
@elylucas  Good call out. I think we should just use GHA so should I comment out the Circle job or actually delete the .circle directory?

Reply:
We can remove it. If we need to get back to it later we can